### DISCUSSION TITLE
Lokking for like minded people!!

### DISCUSSION THREAD

**Discussion by ayan1711** [Cyfrin/foundry-full-course-cu#798] [General]
Hey everyone!!
Looking for like minded people to learn together.

**Comment by Star-light07** [comment]
Hello

**Reply by infinite969** [reply]
hey fren!

**Reply by GenevaRoc** [reply]
Wagwan!!

**Reply by infinite969** [reply]
https://discord.gg/U5quZCPK

**Comment by Amio13** [comment]
here maybe create A DISCORD and send us invite..

**Reply by infinite969** [reply]
I like this idea

**Reply by GenevaRoc** [reply]
How do we connect on the discord?

**Reply by Amio13** [reply]
Once a discord or telegram group is Create and Link dropped here..all interested will join..just few ... quality... quality over quantity

**Reply by Vishal761** [reply]
Yes thats good idea lets make telegram group and meet over there.

**Reply by infinite969** [reply]
hey frens I created a discord server https://discord.gg/U5quZCPK

**Reply by infinite969** [reply]
still putting stuff together, please drop server suggestions in the chat. looking to connect with others learning solidity

**Reply by Amio13** [reply]
Let's discussion starts over in discord then... Join...join and say hi
